### PR TITLE
Project Orion

### PR DESCRIPTION
**Pull request by lngredients**
Added in new surgery room at Exoresearch for Project Orion. Changed Theta project names to Project Orion for guns and armor.

Armor thickness is now only on helmet and armor pieces. Undersuit no longer provides armor values. All armor values were decreased so that max is 65 on main armor piece only for limbs.

Added in Orion Project Race called "Orion" which Admins will have to manually set once proper RP is done ICly to create project Orion. Added in armor race restrictions so it can only be worn once race is set.

Changed Theta Project .DM file name to Orion Project. Changed Project Theta evelope to reflect Project Orion's needs.

Removed the RD closet containing the Theta Project Envelope. Replaced Blank Envelopes in Exo-Research 1 with Project Orion Envelopes

Changed MA3 custom gun to custom MA9 icon.

### CODE REVIEW

**Comment by lngredients** [comment]
If this Clears the CI I'd appreciate a review by someone to see what they think about it.

**Comment by DoctorOctopusTheOctopi** [comment]
@Desolane our Head Mapper and Head Admin has requested the temporary removal of Project Theta/Orion as such I am putting this PR on a DNM until they can give their official verdict.

**Comment by DoctorOctopusTheOctopi** [comment]
After a consult with @BDpuffy420 and understanding on why our Head Boy asked for its removal, I'm removing the DNM Tag. I'll throw up my personal review later and give @Desolane or @caelaislinn a chance to reply too. To confirm that the nerfs/everything is to their standards/what they want.

**Comment by lngredients** [comment]
Oh, last Desolane had told me was to work on Project Orion... Well alright.

**Comment by lngredients** [comment]
CI seems to have passed. Let me know if the Armor values are too much still.

**Comment by lngredients** [comment]
I tried to stick to the Lore as much as possible, for instance the wiki said that the equipment given to the Orion Project was largely unknown except for the M99 sniper. So I made the M98 a carbine for use with the project.

**Comment by lngredients** [comment]
Oh, also I will have to add in the Orion Code Envelope to the UNSC frigate CO quarters and probably the bridge if this does get merged which will be a small addition. It is the other half of the Orion Project Envelope so that the Researcher gives the code phrase to allow for Project Orion to be allowed on board the frigate with some independence. But I was told not to mess with more then one map at a time so that will wait a bit.

**Comment by Flaksim** [comment]
Can I see a screenshot? Of where you added it that is

**Comment by lngredients** [comment]
https://gyazo.com/2af68ab531541f93899721140c90aa1f

**Comment by lngredients** [comment]
Large view
https://gyazo.com/49f8f454eaa28409b836fff2ec3428d4

**Comment by lngredients** [comment]
Not sure what's going on with the desert outpost CI check now

**Comment by lngredients** [comment]
Uh... well it seems to be working now so GG.

**Comment by DoctorOctopusTheOctopi** [comment]
That is a large open space for the lab. Is there any way you could shrink it or fill it with something? It'll look extremely empty in live.

**Comment by lngredients** [comment]
Yeah I will see about filling it wit some other things. I'm a big fan of large open surgery rooms, often things are crammed so tightly together.

**Comment by lngredients** [comment]
https://gyazo.com/bc6735dfc0382d0a1e38431bfca107fd

File: code/game/machinery/nuclear_bomb.dm [hunk 1]
```diff
@@ -476,27 +476,73 @@ var/bomb_set
...
+	We have placed the required assets into a surgery room located west of the Weapons Testing Lab. We feel these items will be sufficient as the final product will be considered a living weapon and is likely to be a highly lethal and decisive utlility on the battlefield.<br><br>\
+	We are expecting great results from this project as it's succesful implementation will immensely bolster our military assets on the ground, when you are done reading this document be sure to burn it in order to eliminate any trace of this project's paper trail. You have full authority to destroy the Project Orion Laboratory should it be discovered, as well as any assets attached to Project Orion.<br><br>\
```
Comment (caelaislinn) [inline code/game/machinery/nuclear_bomb.dm]: *successful

File: code/game/machinery/nuclear_bomb.dm [hunk 2]
```diff
@@ -476,27 +476,73 @@ var/bomb_set
...
+	If you are successful in enabling Project Orion then you will be tasked with procedurally testing Project Orion in live military operations at your descretion.<br><br>\
+	You will have limited authority to request transport for Project Orion utilizing local military vessels in the system via the following clearance phrase.<br><br>\
+	'Omega-Romeo-India-Oscar-November'.<br><br>\
```
Comment (caelaislinn) [inline code/game/machinery/nuclear_bomb.dm]: This should be randomised in the code like traitor secret phrases

File: code/game/machinery/nuclear_bomb.dm [hunk 3]
```diff
@@ -476,27 +476,73 @@ var/bomb_set
...
+	var/obj/item/weapon/paper/orioncode = new(src)
+	orioncode.set_content("<tt><center><b><font color='red'>CONFIDENTIAL: Naval High Command ONLY</font></b>\
+	<h3>Office of Naval Intelligence</h3>\
+	<img src = ntlogo.png>\
+	</center>\
+	<b>FROM:</b> I.N.G. <br>\
+	<b>TO:</b> Research Director of Exo-Research Outpost<br>\
+	<b>SUBJECT:</b>Orion Code<br>\
+	We have tasked our Exo-Research station with a special project the details of which are only revealed on a need-to-know basis. This document exists solely to inform you of the following pass-phrase.<br><br>\
+	'Omega-Romeo-India-Oscar-November'<br><br>\
```
Comment (caelaislinn) [inline code/game/machinery/nuclear_bomb.dm]: Randomised code phrase

File: code/game/machinery/nuclear_bomb.dm [hunk 4]
```diff
@@ -476,27 +476,73 @@ var/bomb_set
...
+	If you are successful in enabling Project Orion then you will be tasked with procedurally testing Project Orion in live military operations at your descretion.<br><br>\
+	You will have limited authority to request transport for Project Orion utilizing local military vessels in the system via the following clearance phrase.<br><br>\
+	'Omega-Romeo-India-Oscar-November'.<br><br>\
+	Any high command aboard local military vessels within the sector should be directed towards documents kept in their personal quarters to verify this code and should allow Project Orion limited use of their facilites and armory during active operations.<br><br>\
+	If the Project Orion subject is designate as AWOL, Unstable, Unusable, or a threat to UNSC or ONI personel you are granted the authority to initate a man hunt or order the execution of the Project Orion Specimen. In these cases all Project Orion assets should be reclaimed if possible, or destroyed on-site ASAP.<br><br>\
+	Do not disappoint us.<br><br><br>\
+	<i>I.N.G.</i></tt>")
```
Comment (caelaislinn) [inline code/game/machinery/nuclear_bomb.dm]: Whats I.N.G. ?

**Review by caelaislinn — CHANGES_REQUESTED** [review]
Code changes look fine. Some minor tweaks requested

**Comment by Flaksim** [comment]
I have performed an extensive remap of exoplanet research in my PR with this change in mind, merge this PR first, then merge mine to ensure the map is up to date.

File: code/game/machinery/nuclear_bomb.dm (hunk 2 above)
Comment (lngredients) [reply code/game/machinery/nuclear_bomb.dm]: Oooh? How do I pull that off, I will look at traitor code!

File: code/game/machinery/nuclear_bomb.dm (hunk 4 above)
Comment (lngredients) [reply code/game/machinery/nuclear_bomb.dm]: It's just an Easter egg for part of my Byond Key and it sort of looks official also. *cute face*

**Pull request closed without merging**